script/.copyrigth_aliases.json

```diff
@@ -0,0 +1,133 @@
+[
```

Comment:
To make this more "parseable" (if for whatever reason we ever want to) shouldn't we make this a object where the `name` is the key for an object with two "sub-keys": `mails` and `authorative_mail`?

Reply:
I think it's a good idea, but I don't want to change it, ever 😄 (We'd have to change the existing copyrite alias in astroid/pylint, it's not worth it imo).

Reply:
Backtracking on that : the current format permit inconsistencies in the json, (see the "bot" it has two entries with 2 sets of ``mails``). I'm going to make a migration tool.

Reply:
This is blocking the release of astroid 2.11, so it's not perfect but I'm going to do that in https://github.com/Pierre-Sassoulas/contributors-txt/issues/3... later.

Reply:
Are you saying you're going to merge this and do a follow-up after `2.11`, or are you moving this from `2.11`?

Either way, I'd say let's move to `2.12`. Doesn't really make sense to do two commits of 100+ lines in quick succession. This is not impacting the performance of `astroid` for now.

Reply:
I was thinking about merging before 2.11 without changing the data structure of the json. It's not really problematic that there are duplication in the json. The fix will be a version upgrade + a migration of the json.

Reply:
That does feel kind of rushed: why would we commit code/data that we know is faulty. As I said, since there is no reason to rush this in the next 24 hours and we'll likely have some more review time while we wait on the last `pylint` `2.13` PRs to be adopted/reviewed I think it's best to make this PR the _big_ copyright PR. That would also help with porting this PR to `pylint` as all open discussions would be solved.

Reply:
Fair enough.

Reply:
Done, and upgraded :)